DOC-2122: Add instruction for activating a newly added user (#2536)
RedisLabs/redislabs-docs: content/rc/security/access-control/access-management.md

| **Billing emails** | Notifications when bills are issued, paid, and so on.<br/>_Note:_ Limited to a single user. |
| **Multi-factor authentication** | Whether MFA is enabled for the member.  This is deactivated when members have not enabled or confirmed MFA in their user profile settings. |

Use the **Add user** button to save your new team member details. 

> Redis will send an activation email to the user once their details are saved. The user must click the activation link in the email before they can sign in to the Redis Cloud admin console.

Reviewer: ```suggestion
Redis will send an activation email to the user once their details are saved. After following the activation link, they can sign in.
```
